release.config.js

```diff
@@ -0,0 +1,41 @@
+const releaseRules = {
+  preset: 'eslint',
+  releaseRules: [
+    { tag: 'Breaking', release: 'major' },
+    { tag: 'Build', release: 'minor' },
+    { tag: 'Chore', release: 'minor' },
+    { tag: 'Fix', release: 'patch' },
+    { tag: 'New', release: 'minor' },
+    { tag: 'Update', release: 'minor' },
+    { tag: 'Upgrade', release: 'minor' }
```

Comment:
The commits should be in this format now, in order to publish a new release based on them.